extensions/sql/asset-index-sql/docs/schema.sql

```diff
@@ -14,29 +14,32 @@
 -- THIS SCHEMA HAS BEEN WRITTEN AND TESTED ONLY FOR POSTGRES
 
 -- table: edc_asset
-CREATE TABLE IF NOT EXISTS edc_asset (
-    asset_id VARCHAR NOT NULL,
-    PRIMARY KEY (asset_id)
-    );
+CREATE TABLE IF NOT EXISTS edc_asset
+(
+    id VARCHAR NOT NULL,
```

Comment:
Some other sql scripts uses the `<entityname>_id` convention, I guess this is needed to permit mapping with jdbc, but wouldn't be better to have the `asset_id` field in the property table called `asset_id_fk`?

Reply:
ok, so the PK would be `asset_id`, and the FK would be `asset_id_fk`

Reply:
yes, sorry for the bad grammar :smiling_face_with_tear: